### DISCUSSION TITLE
Drawing a great circle

### DISCUSSION THREAD

**Discussion by anamitrasikdar** [mtex-toolbox/mtex#1652] [Ask Anything] [answered]
Hello Mtex community,

My question might be silly but somehow I cannot figure the way around the problem.
I have plotted the odf of Muscovite using plotPDF (attached figure).

Now, I want to find out the angle this distribution is making with the E-W of the circle.
My plan was to draw a great circle running along the middle of the distribution and find out the angle of it's pole with the North.

Can anyone help me out on this one?
![untitled](https://user-images.githubusercontent.com/99597265/201592555-d11af2f4-f66a-43f5-80be-6155d716ba5c.jpg)

**Comment by kilir** [comment, marked as answer]
Hi,

here's a way to do what you actually want:
```
%% from orientations
pdf110 = calcDensity(ori * Miller(1,1,0,cs))
%from odf
pdf110 = calcPDF(odf,Miller(1,1,0,cs))

% get eigenvectors of distribution
[egv,eval]=eig(pdf110);
% normal to eogenvector of smallest eigenvalue and z-direction
pxy = cross(egv(3),zvector);
% angle to the yvector
angle(pxy,yvector)
```

In order to draw a circle, you can use `circle(egv(3))` or `plot(egv(3),'circle')`.

Cheers,
Rüdiger

**Reply by anamitrasikdar** [reply]
Thank you Prof. Kilian.
Could you please tell me briefly what is the implication of calculating eigenvectors and taking the smallest eigenvalue?
These lines seems to work for some hkl. For others it is not working (I mean it is not extracting the angle required, to orient the pdf along the E-W).

I just want to point out that MTEX Z = Y and MTEX Y = Z. So, I changed the code block accordingly.

% get eigenvectors of distribution
[egv,eval]=eig(pdf111);
% normal to eogenvector of smallest eigenvalue and y-direction
pxz = cross(egv(3),yvector);
% angle to the zvector
angle(pxz,zvector)

Best regards,
Anamitra

**Reply by zmichels** [reply]
Hi.

Three eigenvectors can be used to describe the 3D distribution of axes/vectors in your texture.
I'm the case of a *perfect* girdle distribution, two eigenvectors would have identical magnitudes/eigenvalues, and the third eigenvector would have a smaller eigenvalues than either of the other two. This 3rd eigenvector is normal to the plane of the girdle. 
If you have a distribution which is girdle-like (but not prefect), two of the eigenvectors should have eigenvalues that are notably larger than the third. The eigenvector with the smallest of the three eigenvalues is normal to a plane that likely fits well with the girdle-y distribution.

Maybe that helps explain why the approach was suggested.

Cheers,
Zach

**Reply by anamitrasikdar** [reply]
Hi Zach,
 
Thank you for clearing it out.

So, in the case of point distribution, the approach can be 
max(pdf), if I understood correctly.

Regards,
Anamitra

**Reply by zmichels** [reply]
Correct.
You might appreciate the following:

Vollmer, F.W., 1990. An application of eigenvalue methods to structural domain analysis. Geological Society of America Bulletin 102, 786-791.

**Reply by anamitrasikdar** [reply]
Thank you Zach.
